### ISSUE TITLE
Add namespace support to Honeycomb helm chart

### ISSUE THREAD

**Issue by kentquirk** [honeycombio/helm-charts#158]
#146 described a problem, and #155 fixed it, but only for the collector helm chart. We should have the same feature in the Honeycomb helm chart.

**Comment by pdeveltere** [comment]
Yes, would be lovely if we could specify the namespace for the honeycomb agent 🙏🏽
